### PR TITLE
i18n(ko-KR): update `typescript.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)

- update `typescript.mdx`
- #6741
- #6702 
- #6687

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

File: src/content/docs/ko/guides/typescript.mdx [hunk 1]
```diff
@@ -218,6 +218,26 @@ type MyAttributes = astroHTML.JSX.ImgHTMLAttributes;
...
 
+### `ComponentProps` 타입
+
+<p>
+  <Since v="4.3.0" />
+</p>
+
+이 타입 내보내기를 사용하면 컴포넌트가 해당 `Props` 타입을 직접 내보내지 않더라도 다른 컴포넌트에서 허용하는 `Props`를 참조할 수 있습니다.
+
+The following example shows using the `ComponentProps` utility from `astro/types` to reference a `<Button />` component's `Props` types:
```
Comment (dreyfus92) [inline src/content/docs/ko/guides/typescript.mdx:229]: This can be removed.

**Review by dreyfus92 — COMMENTED** [review]
Awesome job @jsparkdev, I left you a small suggestion 😁

**Comment by jsparkdev** [comment]
thank you! I fixed it.

**Review by dreyfus92 — APPROVED** [review]
Thank you @jsparkdev, LGTM 😁

**Pull request merged**